css/styles.css

```
@@ -453,33 +449,48 @@ body {
 .field {
   width: 600px;
   height: 60px;
-  background-color: #f2f2f2;
   padding: 0;
   margin: 0;
   padding-left: 22px;
   padding-right: 19px;
-  border-radius: 0px 10px 10px 0px;
-  transform: matrix(-1, 0, 0, 1, 0, 0);
+  border-radius: 10px 0px 0px 10px;
   outline: none;
+  font-size: 18px;
+  line-height: 26px;
+  font-family: PT Sans;
```

А где кавычки и начертание? О_о
Можно кстати поступить проще, заставив элемент наследовать шрифт:
```suggestion
  font-family: inherit;
```

---

уже поправила)